code/__HELPERS/logging/attack.dm

```diff
@@ -82,3 +82,22 @@
 
 	if(message_admins)
 		message_admins("[user ? "[ADMIN_LOOKUPFLW(user)] at [ADMIN_VERBOSEJMP(user)] " : ""][details][bomb ? " [bomb.name] at [ADMIN_VERBOSEJMP(bomb)]": ""][additional_details ? " [additional_details]" : ""].")
+
+/**
+ * Log a combat message in the attack log, multiple targets.
+ *
+ * Arguments:
+ * * atom/user - argument is the actor performing the action
+ * * list/targets - argument is the list of targets of the action
+ * * what_done - is a verb describing the action (e.g. punched, throwed, kicked, etc.)
+ */
+
```

Comment:
```suggestion
 * * what_done - is a verb describing the action (e.g. punched, thrown, kicked, etc.)
 */
```

Reply:
Gonna update the original text as well as I copied this.